Realm/Realm/Linq/SortClause.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using System.Linq;
+using System.Linq.Expressions;
+
+namespace Realms
+{
+    internal class SortClause
+    {
+        public OrderingClause _orderingClause;
+
+        public OrderingClause VisitOrderClause(MethodCallExpression orderClause)
+        {
+            if (orderClause.Method.Name == nameof(Queryable.OrderBy))
```

Comment:
As I have said this is something that needs to be discussed with Jakob, about how exactly this clause needs to look like, especially in regards to isAcending / isReplacing.